services/settings/pkg/config/config.go

```diff
@@ -37,7 +37,7 @@ type Config struct {
 
 	SetupDefaultAssignments bool `yaml:"set_default_assignments" env:"SETTINGS_SETUP_DEFAULT_ASSIGNMENTS;IDM_CREATE_DEMO_USERS" desc:"The default role assignments the demo users should be setup."`
 
-	ServiceAccountIDAdmin string `yaml:"service_account_id_admin" env:"OCIS_SERVICE_ACCOUNT_ID;SETTINGS_SERVICE_ACCOUNT_ID_ADMIN" desc:"The ID of the service account having the admin role. See the 'auth-service' service description for more details."`
+	ServiceAccountIDs []string `yaml:"service_account_ids" env:"SETTINGS_SERVICE_ACCOUNT_IDS;OCIS_SERVICE_ACCOUNT_ID" desc:"The list of all service account IDs. These will be assigned the hidden 'service-account' role See the 'auth-service' service description for more details about service accounts."`
```

Comment:
We need to adjust `SETTINGS_SERVICE_ACCOUNT_ID_ADMIN` in the helmchart. But since service accounts are not released I think we can do this without deprecation.

I wanted the default to be `OCIS_SERVICE_ACCOUNT_ID` if `SETTING_SERVICE_ACCOUNT_IDS` is not set. Would you prefer too have two different envvars and document that `OCIS_SERVICE_ACCOUNT_ID` is only used when the other is not set?

Reply:
See latest commit for what I mean :smile:

Reply:
> We need to adjust SETTINGS_SERVICE_ACCOUNT_ID_ADMIN in the helmchart. But since service accounts are not released I think we can do this without deprecation.

:+1: 



> I wanted the default to be `OCIS_SERVICE_ACCOUNT_ID` if `SETTING_SERVICE_ACCOUNT_IDS` is not set. Would you prefer too have two different envvars and document that `OCIS_SERVICE_ACCOUNT_ID` is only used when the other is not set?

Hm, that also looks clumsy. I guess it would be enough to just add some explicit info to the docstring that `SETTING_SERVICE_ACCOUNT_IDS` can have multiple values while `OCIS_SERVICE_ACCOUNT_ID` can't.